lib/charms/opensearch/v0/constants_charm.py

```diff
@@ -67,9 +67,16 @@
     "Less than 3 cluster-manager-eligible units in this cluster. Add more units."
 )
 PluginConfigError = "Unexpected error during plugin configuration, check the logs"
+PClusterMissingStorageRelations = "Found credentials with missing storage relations."
 BackupSetupFailed = "Backup setup failed, check logs for details"
 BackupRelShouldNotExist = "This unit should not be related to backup relation"
 BackupRelDataIncomplete = "Backup relation data missing or incomplete."
+BackupCredentialIncorrect = "Backup configuration error: bad credentials, permissions, invalid CA, or unsupported configuration."
+BackupCredentialCleanupFailed = (
+    "Failed to remove keystore credentials, repo removal.Please check the logs for more details."
```

Comment:
Missing a space here.
```suggestion
    "Failed to remove keystore credentials, repo removal. Please check the logs for more details."
```
I also suggest changing "repo removal" to something like:
```suggestion
    "Failed to remove keystore credentials or snapshot repository. Please check the logs for more details."
```

Reply:
Done.